test_conformance/spirv_new/test_decorate.cpp

```diff
@@ -303,9 +324,12 @@ int verify_saturated_results(cl_device_id deviceID, cl_context context,
     err = clEnqueueReadBuffer(queue, res, CL_TRUE, 0, out_bytes, &h_res[0], 0, NULL, NULL);
     SPIRV_CHECK_ERROR(err, "Failed to read to output");
 
+    cl_half_rounding_mode half_rounding = get_half_rounding_mode(deviceID);
```

Comment:
Ideally we should only call `get_half_rounding_mode` if `std::is_same<Ti, cl_half>::value`.

Reply:
Thank you! Fixed.
I reran the decorate_* tests on our side using RTZ hardware, and all results passed.